routes/account.js

```diff
@@ -611,6 +784,14 @@ module.exports = function (
 
   if (isProduction) {
     delete routes[0].config.validate.payload.preVerified
+
+    for (var i = 0; i < routes.length; ++i) {
+      var route = routes[i];
+      if (route.path === '/account/lock') {
+        routes = routes.splice(i, 1)
```

Comment:
This doesn't look quite right to me.  IIUC `splice` modifies the array in-place and returns the removed elements...so would this actually leave you with *only* the `/account/lock` route in production?

Untestable only-on-in-production things are terrifying, but necessary in this case.  I wonder if we can factor this out into a `removeRoute` helper function that is testable, and a simple call to it thats "obviously correct" in production.